kernel/kernel-config-4.9.x

```diff
@@ -1730,6 +1733,15 @@ CONFIG_RTC_MC146818_LIB=y
 # CONFIG_SYNC_FILE is not set
 # CONFIG_AUXDISPLAY is not set
 # CONFIG_UIO is not set
+CONFIG_VFIO_IOMMU_TYPE1=y
+CONFIG_VFIO_VIRQFD=y
+CONFIG_VFIO=y
+CONFIG_VFIO_NOIOMMU=y
```

Comment:
@amshinde What happens once we move to QEMU 2.9. Do we need to disable this?

Reply:
Yes @mcastelino, we would need to disable this.